.changeset/mighty-shirts-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Remove localStorage synchronization on storage events in Safari iframes. See [GitHub PR #8408](https://github.com/firebase/firebase-js-sdk/pull/8408).
```

Comment:
FYI the changeset tool automatically links to the PR that the changeset was added in when creating the Version Packages PR, but this does make the copy-paste into release notes easier.